kivy/tests/test_mouse_hover_event.py

```diff
@@ -1,6 +1,11 @@
+import pytest
+
+from kivy import platform
 from kivy.tests.common import GraphicUnitTest
 
 
+@pytest.mark.skipif(platform == 'win',
+                    reason='Causes test_mouse_multitouchsim.py to fail on CI.')
```

Comment:
```suggestion
@pytest.mark.skipif(
  platform == 'win' and 'CI' in os.environ,
  reason='Causes test_mouse_multitouchsim.py to fail on CI.'
)
```

I'd rather it's only [skipped on CI](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables).